include/NovelRT/Physics/Box2d/Components/PhysicsBody.h

```diff
@@ -0,0 +1,24 @@
+#ifndef NOVELRT_PHYSICS_BOX2D_PHYSICSBODY_H
+#define NOVELRT_PHYSICS_BOX2D_PHYSICSBODY_H
+
+#include <box2d/b2_body.h>
+
+namespace NovelRT::Physics::Box2d
+{
+    struct PhysicsBody
+    {
+        b2Body* body;
+
+        static const PhysicsBody DeletedBodyState;
```

Comment:
I made this static const because i need to give a state that signals a deleted component tothe buffer and i did it the way i would do it in C#. Although it works, i am not sure if this is the best way to do it.